merkle/log_proofs.go

```diff
@@ -90,37 +90,29 @@ func CalcConsistencyProofNodeAddresses(snapshot1, snapshot2, treeSize int64) ([]
 	return snapshotConsistency(snapshot1, snapshot2, treeSize)
 }
 
-// snapshotConsistency does the calculation of consistency proof node addresses between
-// two snapshots. Based on the C++ code used by CT but adjusted to fit our situation.
+// snapshotConsistency does the calculation of consistency proof node addresses
+// between two snapshots in a bigger tree of the given size.
```

Comment:
We might want to remove 'snapshot' terminology at some point as it was carried over from C++.

What about just saying "between two tree sizes" and noting that snapshot2 must be >= snapshot1.

Reply:
Good idea. Will remove this terminology in a follow-up PR.